BookingQueueSubscriber/BookingQueueSubscriber.Services/UserCreationAndNotification.cs

```diff
@@ -89,16 +89,28 @@ private async Task<User> CreateUserAndSendNotificationAsync(HearingDto hearing,
                     await _bookingsApiClient.UpdatePersonUsernameAsync(participant.ContactEmail, participant.Username);
                 }
             }
+            
 
             if (user != null)
             {
+                var userPassword = user.Password;
                 if (_featureToggles.UsePostMay2023Template() && participant.IsIndividual())
                 {
-                    await _notificationService.SendNewUserWelcomeEmail(hearing, participant);
-                    // await _notificationService.SendNewUserAccountDetailsEmail(hearing, participant, user.Password);
-                    // when VIH-9899 is implemented, send the 'New' NewUserAccountNotification here and put the original in the else block
+                    if (!string.IsNullOrEmpty(userPassword))
+                    {
+                        await _notificationService.SendNewUserWelcomeEmail(hearing, participant);
+                        await _notificationService.SendNewUserSingleDayHearingConfirmationEmail(hearing, participant, userPassword);
+                    }
+                    else
+                    {
+                        await _notificationService.SendExistingUserSingleDayHearingConfirmationEmail(hearing, participant);
+                    }
+                    
+                }
+                else
+                {
+                    await _notificationService.SendNewUserAccountNotificationAsync(hearing.HearingId, participant, user.Password);
```

Comment:
might be worth changing this from `user.Password` to `userPassword` for consistency with the above

Reply:
done